### PR TITLE
PLuG user identification docs improvements

### PR DESCRIPTION
**Pull request by kunal-mohta-99**
## Summary
This change improves the PLuG Web SDK's user identification documentation by adding a screenshot and a new section for "Changing the user identity" which adds some frequently used code snippets.

## Craftsmanship, Integrity, and Devil’s Advocacy

- [ ] Testing: Negative test cases: null or default values, crash and fault injection tests
- [ ] Testing: Boundary conditions: rolling upgrades, denial-of-service, etc.
- [ ] Testing: Fixing a few existing — flaky or permanently-broken — test cases
- [ ] Observing: Detailed error codes so machines can understand
- [ ] Observing: Adding superior metrics for future debugging
- [ ] Observing: Tracing the hairiest pathways for field serviceability
- [ ] Training: KnowledgeOps update? So AI always works.

### CODE REVIEW

**Review by bc-devrev — CHANGES_REQUESTED** [review]
Please implement the diagram in Mermaid. https://buildwithfern.com/learn/docs/content/write-markdown#diagrams

File: fern/docs/pages/sdks/web/user-identity.mdx [hunk 1]
```diff
@@ -208,3 +211,60 @@ window.plugSDK.updateIdentity({
...
+  app_id: appId,
+
+  // With session token
+  session_token: sessionToken,
+
+  // Or without session token
+  identity: {},
+});
+```
+<Callout intent="note">
```
Comment (bc-devrev) [inline fern/docs/pages/sdks/web/user-identity.mdx:243]: This is too much for a callout. A section would be a more suitable container.

**Review by bc-devrev — APPROVED** [review]
